### ISSUE TITLE
Yet more HD parrots to add

### ISSUE THREAD

**Issue by uberchemist** [jmhobbs/cultofthepartyparrot.com#303]
Time for another deluge of HD parrots! Here's everything new since my last post, issue #300. I'll also post any new parrots to my [Imgur account](https://uberchemist.imgur.com/all) as I make them, if you'd rather not wait. I'm working on replacing many of the lower quality parrot gifs with HD quality.
[Slow Parrot](https://imgur.com/FXLXzcH): ![slow](https://user-images.githubusercontent.com/46827621/52070579-00990200-2536-11e9-8d1e-611335d0f2ce.gif) (spent a lot of time on this one, pretty happy with how it turned out)
[Angry Parrot](https://imgur.com/YLfPlkc) (updated): ![angry](https://user-images.githubusercontent.com/46827621/52136425-e70ebd80-25fc-11e9-87ba-0cbe4803bcb3.gif)
[Confused Parrot](https://imgur.com/lZ2RnJ9): ![confused](https://user-images.githubusercontent.com/46827621/52070768-6e452e00-2536-11e9-8ffb-9aede3ba9b84.gif)
[Thumbs Up Parrot](https://imgur.com/5XbEwAF): ![thumbsup](https://user-images.githubusercontent.com/46827621/52070950-d431b580-2536-11e9-8c2f-a74c46669148.gif)
[Cop Parrot](https://imgur.com/NNwM7gC): ![cop](https://user-images.githubusercontent.com/46827621/52071536-15769500-2538-11e9-82e3-9277108bd349.gif)
*New Edit* [60fps Parrot](https://imgur.com/OGFMziz): ![60fps](https://user-images.githubusercontent.com/46827621/52219894-6ea13a00-2852-11e9-803a-af3f82eff0c3.gif)

### **And now for all the new flag parrots!**

[France](https://imgur.com/W21eSho): ![france](https://user-images.githubusercontent.com/46827621/52071921-d6950f00-2538-11e9-958e-b473335bb022.gif)
[Germany](https://imgur.com/d1FBtEU): ![germany](https://user-images.githubusercontent.com/46827621/52071931-db59c300-2538-11e9-9c48-b62e794928d3.gif)
[Spain](https://imgur.com/XcCSaI4): ![spain](https://user-images.githubusercontent.com/46827621/52071941-deed4a00-2538-11e9-92b9-118d52cb4ad0.gif)
[Portugal](https://imgur.com/pLwqFDf): ![portugal](https://user-images.githubusercontent.com/46827621/52071951-e3b1fe00-2538-11e9-9857-13d4de839fe2.gif)
[Europe](https://imgur.com/aGjdJ34): ![europe](https://user-images.githubusercontent.com/46827621/52071957-e876b200-2538-11e9-8e40-a4fd1140991b.gif)
[Ireland](https://imgur.com/2AdmcVy): ![ireland](https://user-images.githubusercontent.com/46827621/52071964-eca2cf80-2538-11e9-8b0b-49303cbafd50.gif)
[Japan](https://imgur.com/os7S2ca): ![japan](https://user-images.githubusercontent.com/46827621/52071969-f0ceed00-2538-11e9-83ba-84adfe318c52.gif)
[Philippines](https://imgur.com/7ILbgdA): ![philippines](https://user-images.githubusercontent.com/46827621/52071979-f4627400-2538-11e9-8600-9366c0e5b13b.gif)
[Sweden](https://imgur.com/OjZDrEu): ![sweden](https://user-images.githubusercontent.com/46827621/52071988-f9bfbe80-2538-11e9-8845-e8abfe790853.gif)
[Australia](https://imgur.com/0HM5Xxg): ![australia](https://user-images.githubusercontent.com/46827621/52072006-047a5380-2539-11e9-8897-7f38d2d2a5c0.gif)
[New Zealand](https://imgur.com/qvHQE8a): ![newzealand](https://user-images.githubusercontent.com/46827621/52072017-08a67100-2539-11e9-8998-89802a951fd5.gif)
[India](https://imgur.com/0xirfaU): ![india](https://user-images.githubusercontent.com/46827621/52072026-0c39f800-2539-11e9-980f-3a9dea8d9ebd.gif)
[Pakistan](https://imgur.com/pQjMMcG): ![pakistan](https://user-images.githubusercontent.com/46827621/52072033-10feac00-2539-11e9-9f17-a93d73e6fc2a.gif)
[South Africa](https://imgur.com/pPjikBU): ![southafrica](https://user-images.githubusercontent.com/46827621/52072038-152ac980-2539-11e9-971f-65e54bce29c9.gif)
[Dominica](https://imgur.com/otJKtfc): ![dominica](https://user-images.githubusercontent.com/46827621/52072047-1a881400-2539-11e9-8aa3-874334a78033.gif) (in case you're wondering about this one, their flag has a PURPLE PARROT on it!)

**Comment by uberchemist** [comment]
Hey there, I was actually thinking of redoing that myself, it's a bit too fuzzy. And yes, hard to notice in Slack. I'll fix and repost soon.

**Comment by francoislg** [comment]
This slow parrot is awesome!

So awesome that I tried to put it back to normal speed:

![60fpsparrot](https://user-images.githubusercontent.com/8355585/52123898-7617d780-25f5-11e9-8f6e-ed960734e231.gif)

What about `60fpsparrot` or `smoothparrot`

**Comment by jmhobbs** [comment]
These are so great @uberchemist!   I'll roll these in shortly, I'm in the middle of splitting out the flag parrots so I can drop all of those in with that PR.

**Comment by uberchemist** [comment]
Angry parrot has been updated in the original post. It's still a little hard to see the eyebrows in Slack, but it's crisper, so that helps a bit. The only way to make them more visible would be to go full Groucho Marx (which should probably be a parrot anyway)!

**Comment by uberchemist** [comment]
> 
> 
> This slow parrot is awesome!
> 
> So awesome that I tried to put it back to normal speed:
> 
> ![60fpsparrot](https://user-images.githubusercontent.com/8355585/52123898-7617d780-25f5-11e9-8f6e-ed960734e231.gif)
> 
> What about `60fpsparrot` or `smoothparrot`

Funny! It's still a bit faster than the normal speed, plus the colors cycle twice as fast, so perhaps `epilepsyparrot` is more appropriate! jk...
I do like the `60fpsparrot` concept. I'll mess around with it myself and see if I can make it work.

**Comment by cavpollo** [comment]
@uberchemist +1000 for a Groucho Marx one xD

**Comment by uberchemist** [comment]
@francoislg I've added the 60fps parrot (edit in original post above)! I switched the color cycling back to normal and tried to tone down the speed just a bit. It's not perfect with the original parrot speed, but I compared both faster and slower versions and decided that the slower one allows you to really soak in the glorious 60fps-ness... which, of course, is not an accurate name. "40fps parrot" just doesn't sound as cool!
